cloud/kubernetes/README.md

```diff
@@ -171,6 +168,28 @@ $ kubectl certificate deny default.node.cockroachdb-0
 certificatesigningrequest "default.node.cockroachdb-0" denied
 ```
 
+Once all the pods have started, to initialize the cluster run:
+```shell
+kubectl create -f cluster_init_secure.yaml
+```
+
+This will create a CSR called "default.client.root", which you can approve by
+running:
+```shell
+kubectl certificate approve default.client.root
+```
+
+To confirm that it's done, run:
+```shell
+kc get job cluster-init-secure
```

Comment:
I guess `kc` is your local alias for `kubectl`?

Reply:
Darn, I missed one. Yes, it is.